api/config_entry_gateways_test.go

```diff
@@ -163,11 +170,20 @@ func TestAPI_ConfigEntries_IngressGateway(t *testing.T) {
 			readIngress.Listeners[0].Services[0].Partition = ""
 
 			require.Equal(t, ingress1.Listeners, readIngress.Listeners)
+			require.Equal(t, ingress1.Listeners[0].Services[0].MaxConnections,
+				readIngress.Listeners[0].Services[0].MaxConnections)
+			require.Equal(t, ingress1.Listeners[0].Services[0].MaxConcurrentRequests,
+				readIngress.Listeners[0].Services[0].MaxConcurrentRequests)
+			require.Equal(t, ingress1.Listeners[0].Services[0].MaxPendingRequests,
+				readIngress.Listeners[0].Services[0].MaxPendingRequests)
```

Comment:
My understanding of `require.Equal` is that it's a deep comparison by value, so I think the new lines here are redundant.